draft-yasskin-http-origin-signed-responses.md

```diff
@@ -331,7 +331,7 @@ Accept: */*
 
 HTTP/1.1 200
 Content-Type: text/html
-MI: mi-sha256=20addcf7368837f616d549f035bf6784ea6d4bf4817a3736cd2fc7a763897fe3
```

Comment:
https://martinthomson.github.io/http-mice/draft-thomson-http-mice.html#mi-header-field-parameters says that the value is base64url encoding [[RFC7515](https://tools.ietf.org/html/rfc7515)], which does not have padding '=' characters.

Reply:
@irori thx++ :D